translations/messages/src/components/search/searchbar.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "searchbar.search",
+    "defaultMessage": "Search"
+  },
+  {
+    "id": "searchbar.forsale",
+    "defaultMessage": "For Sale"
+  },
+  {
+    "id": "searchbar.newListings",
+    "defaultMessage": "New Listings"
+  },
+  {
+    "id": "searchbar.delasNearMe",
+    "defaultMessage": "Deals Near Me"
+  }
+]
```

Comment:
super nit: add newline

Reply:
I think this file is automatically created by the translation plugin. Will add the new line anyway :)